util/lambda_trigger/lambda_trigger.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"encoding/json"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+// ReleaseEvent event body
+type ReleaseEvent struct {
+	Product string `json:"product"`
+	Version string `json:"version"`
+	SHASUM  string `json:"shasum"`
+}
+
+func shouldTriggerWorkflow(product string) bool {
+	supportedProducts := []string{"vault", "consul", "nomad", "terraform", "packer"}
+
+	for _, p := range supportedProducts {
+		if p == product {
+			return true
+		}
+	}
+
+	return false
+}
+
+func triggerGithubWorkflow(event *ReleaseEvent) error {
+	githubToken := os.Getenv("GITHUB_TOKEN")
+	workflowEndpoint := "https://api.github.com/repos/hashicorp/homebrew-tap/dispatches"
```

Comment:
Just to make it easier for ourselves in six months, should we throw a comment here with a link to the GH docs for this endpoint and body structure?

Reply:
Yeah sure